Normalise more identifier names

Joehuu opened the pull request:
Continuing from https://github.com/ppy/osu/pull/16266.

Joehuu committed:
Bump identifier typo inspection and fix remaining identifier names

bdach commented on osu.Desktop/Windows/WindowsKey.cs:
```diff
@@ -14,6 +14,8 @@ internal class WindowsKey
         private const int wh_keyboard_ll = 13;
         private const int wm_keydown = 256;
+
+        // ReSharper disable once IdentifierTypo
```
Not sure how others feel but I think that for platform-native classes like this it's better to just disable typo inspections file-wide rather than play whack-a-mole locally, as they generally tend to have obscure names from native APIs that may spuriously trigger the inspection at some point.

peppy replied:
Agree

bdach commented on osu.Game/Online/API/Requests/GetUserRecentActivitiesRequest.cs:
```diff
@@ -22,6 +22,8 @@ public GetUserRecentActivitiesRequest(long userId, int page = 0, int itemsPerPag
     {
         Achievement,
+
+        // ReSharper disable once IdentifierTypo
```
Similar as above, I'd just exclude the entire enum from the inspection.

Joehuu committed:
Increase scope of identifier typo disables on special cases

peppy commented:
![20211230 143331 (JetBrains Rider)](https://user-images.githubusercontent.com/191335/147724365-696931e8-0f70-49ea-a2d8-2556605a9ad6.png)

How are none of these coming up on CI? I get them locally. Most seem valid.

peppy commented on osu.Desktop/Windows/WindowsKey.cs:
```diff
@@ -2,10 +2,12 @@
 
 namespace osu.Desktop.Windows
 {
+    [SuppressMessage("ReSharper", "IdentifierTypo")]
```
We generally use comments instead (above the `namespace` line):

`// ReSharper disable IdentifierTypo`

Joehuu replied:
Does this also apply to the enum case? If so, I would have to add a `ReSharper restore` which is a first in this project.

<img width="355" alt="Screen Shot 2021-12-30 at 10 35 27 PM" src="https://user-images.githubusercontent.com/35318437/147807623-8e1aa227-ae53-452c-8723-d4324871222f.png">

peppy replied:
You can use ` // ReSharper disable once IdentifierTypo` here to disable once. Either above the line of above the `enum` definition.

Joehuu replied:
The last review wanted the whole enum to be disabled, see https://github.com/ppy/osu/pull/16273#discussion_r776220493.

bdach replied:
If it's not feasible to disable the inspection on the entire enum with a single line comment than I'm fine with disabling once on the item itself.

Joehuu commented:
> How are none of these coming up on CI? I get them locally. Most seem valid.

Weird, I'm not getting those locally.

peppy commented:
You may have added some words to your local dictionary which need to be added to the "team shared" (aka dotsettings) dictionary.

Joehuu commented:
I'm sure I didn't. Even on another computer with Rider newly installed, it doesn't show with this branch checked out.

peppy commented:
The weird part is that they don't come up on CI.

peppy commented:
If we want to get the fixes in this PR merged, may I suggest turning off the inspection change for now? As it stands they can't be merged as it causes many warnings locally (and we have not yet figured why those don't show on CI).

bdach commented:
I suppose. I'd much rather have CI check that going forward but seeing that the inspection doesn't look to be enforceable there then better this than nothing I guess.

Joehuu committed:
Merge remote-tracking branch 'upstream/master' into fix-remaining-identifier-names

Joehuu committed:
Fix missed identifier typos

Joehuu committed:
Use comments to disable identifier typos instead

Joehuu committed:
Revert identifier typo bump

Joehuu commented:
I've fixed my local settings, but still cannot get that `Framerate` case shown in the screenshot. I've ignored that and just fixed the ones I've seen locally.

peppy committed:
Remove names from user dictionary

peppy commented:
With the inspection enabled I still have 204 warnings. Let's go with these fixes for now and figure out the remaining as a follow-up.
